docs/guides/file_sharing/nfsserver.md

```diff
@@ -0,0 +1,185 @@
+* `*` for everybody.
+
+Multiple clients can be specified on the same line, separated by a space.
+
+### Permissions on resources
+
+There are two types of permissions:
+
+* `ro` : readonly;
+* `rw` : modification.
+
+If no right is specified, then the right applied will be read-only.
+
+By default, client user UIDs and GIDs are conserved (except for `root`).
+
+To force the use of a UID or GID other than that of the user writing the resource, specify the `anonuid=UID` and `anongid=GID` options, or give `anonymous` access to the data with the `all squash`option.
+
+!!! warning "warning" 
+
+    There is a parameter, `no_root_squash`, which identifies the client root user as the server root user. This parameter can be dangerous for system security.
+
+By default, the `root_squash` parameter is activated (even if not specified), identifying `root` as an `anonymous` user.
+
+### Case studies
+
+* `/share client(ro,all_squash)`
+Client users have read-only access to resources and are identified as anonymous on the server.
+
+* `/share client(rw)`
+Client users can modify resources and keep their UID on the server. Only `root` is identified as `anonymous`.
+
+* `/share client1(rw) client2(ro)`
+Users on client workstation 1 can modify resources, while those on client workstation 2 have read-only access.
+UIDs are kept on the server, and only `root` is identified as `anonymous`.
+
+* `/share client(rw,all_squash,anonuid=1001,anongid=100)`
+Client1 users can modify resources. Their UID is changed to `1001` and their GID to `100` on the server.
+
+### The `exportfs` command
+
+The `exportfs` (exported file systems) command is used to manage the table of local files shared with NFS clients.
+
+```
+exportfs [-a] [-r] [-u share_name] [-v]
+```
+
+| Options         | Description                               |
+| --------------- | ----------------------------------------- |
+| `-a`            | Enable NFS shares                         |
+| `-r`            | Apply shares from the `/etc/exports` file |
+| `-u share_name` | Disables a given share                    |
+| `-v`            | Displays the list of shares               |
+
+### The `showmount` command
+
+The `showmount` command is used to monitor clients.
+
+```
+showmount [-a] [-e] [host]
+```
+
+| Options | Description                               |
+| ------- | ----------------------------------------- |
+| `-e`    | Displays shares on the designated server  |
+| `-a`    | Displays all current shares on the server |
+
+This command also determines whether the client workstation has authorization to mount shared resources.
+
+!!! note "note"
+
+    `showmount`sorts and deletes duplicates in the results, so it's impossible to determine whether a client has made multiple mounts of the same directory.
+
+## Client configuration
+
+Access to shared resources on an NFS server is done by mount point on the client.
```

Comment:
?

Reply:
I don't know how to explain that in english.
It means you have to create a mount point on the client side to access to the data on server.

Reply:
Shared resources on an NFS server are accessible through the mount point on the client.

Reply:
through a mount point
or
through the mount point